packages/toolpad-core/src/DashboardLayout/DashboardLayout.test.tsx

```diff
@@ -63,30 +63,34 @@ describe('DashboardLayout', () => {
         title: 'Analytics',
       },
       {
+        segment: 'reports',
         title: 'Reports',
         icon: <BarChartIcon />,
         children: [
           {
+            segment: 'sales',
             title: 'Sales',
             icon: <DescriptionIcon />,
           },
           {
+            segment: 'traffic',
             title: 'Traffic',
             icon: <DescriptionIcon />,
           },
         ],
       },
       {
+        segment: 'integrations',
         title: 'Integrations',
         icon: <LayersIcon />,
       },
     ];
 
     const user = userEvent.setup();
     render(
-      <NavigationContext.Provider value={NAVIGATION}>
+      <AppProvider navigation={NAVIGATION}>
```

Comment:
Not important but for consistency I guess we could do the same with the `BrandingContext` in this file?